assets/js/components/KeyMetrics/MetricsSelectionPanel/Footer.js

```diff
@@ -70,9 +70,7 @@ export default function Footer( {
 			KEY_METRICS_SELECTED
 		)
 	);
-	const keyMetricsSettings = useSelect( ( select ) =>
```

Comment:
The old error text shown inline, is replaced with new better looking UI error component. It is not guarded behind feature flag since it is a change to the selection panel footer, which is global component, not having logic related to the ACR, and new error component looks better